exercises/concept/elyses-enchantments/enchantments.spec.js

```diff
@@ -189,13 +189,13 @@ describe('make the top card disappear', () => {
   });
 
   test('remove two cards from the bottom', () => {
-    const stack = [1, 2, 3];
+    let stack = [1, 2, 3];
 
-    removeItemFromTop(stack);
-    removeItemFromTop(stack);
+    stack = removeItemAtBottom(stack);
+    stack = removeItemAtBottom(stack);
 
     const expected = [3];
-    expect(expected).toStrictEqual(expected);
+    expect(stack).toStrictEqual(expected);
```

Comment:
Instead of changed the top part, only the last line should be changed (which you did, thanks!)